tracer/src/Datadog.Trace/AppSec/Waf/Initialization/WafConfigurator.cs

```diff
@@ -186,29 +186,44 @@ internal UpdateResult Update(IntPtr wafBuilderHandle, ConfigurationState configu
                 result.ReportedDiagnostics.Rest.Warnings is { Count: > 0 })
             {
                 var diags = result.ReportedDiagnostics;
-                var sb = StringBuilderCache.Acquire();
                 DumpStatsMessages(ref diags.Rules);
                 DumpStatsMessages(ref diags.Rest);
 
+                if (diags.HasErrors)
+                {
+                    // This message should go to telemetry logs only, skipping the regular logs
```

Comment:
Just to make it clear that's _not_ what's happening 😅 
```suggestion
                    // TODO: This message should go to telemetry logs only, skipping the regular logs
```

Reply:
Yeah, I just want to make it super clear that this is saying what _should_ happen, not what _is_ happening